build.sbt

```diff
@@ -30,11 +30,11 @@ lazy val chronoscala = crossProject(JSPlatform, JVMPlatform)
 
     scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature"),
 
-    scalacOptions in (Compile, doc) ++= {
+    Compile / doc /scalacOptions ++= {
```

Comment:
method in in trait ScopingSetting is deprecated (since 1.5.0): `in` is deprecated; migrate to slash syntax - https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash